### PR TITLE
Lum 1124 jenkins job

### PR DESCRIPTION
**Pull request by Blackbaud-ChristopherCotar**
@blackbaud/constituent-service 

Not done yet, turns out this branch is publishing an empty jar for the cli, need to configure the maven publishing better.  The copying does work however...

P.S.  I made a team in Github, let's see if it works

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,6 +19,7 @@ task copyCliJarRename(type: Copy) {
 }
 
 copyCliJarRename.dependsOn codegen
+publish.dependsOn copyCliJarRename
```
Comment (Blackbaud-RyanMcKay) [inline build.gradle:23]: consider type: publish

File: build.gradle [hunk 2]
```diff
@@ -19,6 +19,7 @@ task copyCliJarRename(type: Copy) {
...
 
 def artifactFile = new File("${buildDir}/swagger-codegen-cli-${version}.jar")
```
Comment (Blackbaud-RyanMcKay) [inline build.gradle]: use groovy file

File: build.gradle [hunk 3]
```diff
@@ -19,6 +19,7 @@ task copyCliJarRename(type: Copy) {
...
```
Comment (Blackbaud-RyanMcKay) [inline build.gradle:26]: try moving publication out of ext and/or build/lib for jar placement

File: build.gradle [hunk 4]
```diff
@@ -19,6 +19,7 @@ task copyCliJarRename(type: Copy) {
 }
```
Comment (Blackbaud-RyanMcKay) [inline build.gradle]: maven clean yo

**Comment by Blackbaud-ChristopherCotar** [comment]
@Blackbaud-RyanMcKay Merged master from forked project.  Build is green and publishing.

**Pull request merged**